circleci/images/Makefile

```diff
@@ -0,0 +1,123 @@
+# config variable
+DOCKER_REPO=citus
+
+# auto generated variables
+ifeq ($(RELEASE),)
+	TAG_SUFFIX := -dev$(shell date +'%Y%m%d%H%M')
```

Comment:
would it make sense to check if RELEASE is set here instead of the other way around so that we don't go into the else block by accident. 

Would this work:
```
ifdef RELEASE
 TAG_SUFFIG := 
else
  TAG_SUFFIX := -dev$(shell date +'%Y%m%d%H%M')
endif
```

Reply:
will try